vsts/pipelines/nightly.yml

```diff
@@ -51,12 +47,7 @@ jobs:
     name: OryxLinux
   steps:
   - script: |
-      echo "##vso[task.setvariable variable=BuildBuildImages;]false"
-      echo "##vso[task.setvariable variable=BuildRuntimeImages;]false"
       echo "##vso[task.setvariable variable=TestIntegration;]true"
-      echo "##vso[task.setvariable variable=PushBuildImages;]false"
```

Comment:
We need to be consistent. Either we specify all the variables with `false` and `true` or just have `true`. I saw a mix of these in one job and not the other. We could just agree that unless explicitly specified, its always `false`

Reply:
I agree with you, we should write explicitly IMO